### ISSUE TITLE
Whitening in mtcnn.py

### ISSUE THREAD

**Issue by AGenchev** [timesler/facenet-pytorch#146]
in mtcnn.py the function prewhiten is not called.
I wonder then whether the returned images are in the same form the pre-trained models expect.
There is function fixed_image_standardization, but it does not perform normalization, only scaling.

**Comment by dazednconfusing** [comment]
I also have this question. Should we manually prewhiten our images?
